### PR TITLE
Plans: Update feature list and FAQs with premium theme info

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Re-add "Premium themes" as items to the plan features list on the Plans page
* Update copy for themes FAQ on the plans page

**Before**

Screen Shot 2021-12-09 at 10 19 37 AM
Screen Shot 2021-12-09 at 10 19 43 AM

**After**

Screen Shot 2021-12-09 at 10 17 12 AM
Screen Shot 2021-12-09 at 10 16 21 AM

#### Testing instructions

* Switch to this PR, add `?flags=themes/premium` to the URL to enable the flag
* Visit the Plans page for sites of different types; free, personal, premium, business, ecomm, Jetpack, etc.
* "Premium themes" should appear before "Advanced design customization" in the features list, with a tooltip
* Scrolling down a bit, the FAQ for "Can I install my own themes?" should mention premium themes as well
* These elements should not be visible without the themes/premium flag.
* The copy should match the copy suggested in the marketing spreadsheet.

Related to #58780

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -303,10 +303,10 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_PREMIUM_THEMES ]: {
 		getSlug: () => FEATURE_PREMIUM_THEMES,
-		getTitle: () => i18n.translate( 'Unlimited premium themes' ),
+		getTitle: () => i18n.translate( 'Premium themes' ),
 		getDescription: () =>
 			i18n.translate(
-				'Unlimited access to all of our advanced premium themes, including designs specifically tailored for businesses.'
+				'Access to all of our advanced premium theme templates, including templates specifically tailored for businesses.'
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:308]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **31** times:
`translate( 'Unlimited access to all of our advanced premium theme templates, including templates specifically tailored for businesses.' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/feature-list-with-premium-themes)

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 2]
```diff
@@ -17,6 +17,24 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<a href="https://wordpress.com/help" target="_blank" rel="noopener noreferrer" />
 		);
 
+	const themesAnswer = isEnabled( 'themes/premium' )
+		? translate(
+				"Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like." +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/wpcom-faq.jsx:22]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Yes! With the WordPress.com Business or eCommerce plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/feature-list-with-premium-themes)

**Comment by mreishus** [comment]
* There are some typescript errors

The compact function doesn't expect a null to be passed in:
https://github.com/Automattic/wp-calypso/blob/b303ecc52f0aa08c5697be233d0de154bc005d9e/packages/calypso-products/src/plans-list.tsx#L201-L203
I suspect adding null to the compact() definition will be fine. *(False is there twice, maybe whoever added that definition was thinking null when they typed the second false)*

* ~~I can't seem to test with `http://calypso.localhost:3000/plans/<site>?flag=themes/premium` - I never see the new entries. Maybe this is a limitation of the flag system?~~
  * My mistake. I was using `flag` instead of `flags`. After fixing this, I see the new entries fine.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7044968

Thank you @sixhours for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7044968).

**Review by mreishus — APPROVED** [review]
Retested; looks good with both flag on and off. :+1:

**Pull request merged**